specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -1564,922 +2662,1215 @@
         },
-        "hoursOfIce": {
-          "type": "number",
-          "format": "float",
-          "description": "Hours of ice."
+        "isSubtropical": {
+          "type": "boolean",
+          "description": "True when the depression is classified as a subtropical cyclone."
         },
-        "cloudCover": {
+        "govId": {
           "type": "integer",
           "format": "int32",
-          "description": "Percent representing cloud cover."
+          "description": "Government storm ID. This will match the depression number."
         }
       }
     },
-    "WeatherAlongRouteSummary": {
-      "description": "Short summary of the weather along the route.",
-      "type": "object",
-      "properties": {
-        "iconCode": {
-          "$ref": "#/definitions/IconCode"
-        },
-        "hazards": {
-          "$ref": "#/definitions/WeatherHazards"
-        }
+    "basinId": {
+      "type": "string",
+      "description": "Basin identifier",
+      "enum": [
+        "AL",
+        "EP",
+        "SI",
+        "NI",
+        "CP",
+        "NP",
+        "SP"
+      ],
+      "x-ms-enum": {
+        "name": "basinId",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "AL",
+            "description": "Atlantic"
+          },
+          {
+            "value": "EP",
+            "description": "Eastern Pacific"
+          },
+          {
+            "value": "SI",
+            "description": "Southwest Indian Ocean"
+          },
+          {
+            "value": "NI",
+            "description": "North Indian Ocean"
+          },
+          {
+            "value": "CP",
+            "description": "Central Pacific"
+          },
+          {
+            "value": "NP",
+            "description": "Northwest Pacific"
+          },
+          {
+            "value": "SP",
+            "description": "South Pacific"
+          }
+        ]
       }
     },
-    "WeatherAlongRouteResponse": {
+    "StormSearchResult": {
       "type": "object",
-      "description": "This object is returned from a successful Weather Along Route.",
+      "description": "Search government-issued storms",
       "properties": {
-        "summary": {
-          "readOnly": true,
-          "$ref": "#/definitions/WeatherAlongRouteSummary"
-        },
-        "waypoints": {
-          "readOnly": true,
+        "results": {
           "type": "array",
-          "description": "Data for each waypoint returned in the same order as specified in the request.",
           "items": {
-            "$ref": "#/definitions/WeatherWaypoint"
-          }
+            "$ref": "#/definitions/StormSearch"
+          },
+          "description": "Search government-issued storms"
         }
       }
     },
-    "WeatherAlongRoutePrecipitation": {
-      "description": "Precipitation forecast of the weather along the route.",
+    "StormSearch": {
```

Comment:
Copied from APIVIew:
Since `StormSearchResult` is taken by the parent object, this should be `StormSearchResultItem`. Unfortunately, `StormSearch` implies that it is some sort of request body.

Reply:
Thanks for the suggestion! I have updated the name.